packages/@azure/servicebus/data-plane/test/README.md

```diff
@@ -52,9 +51,46 @@
     TOPIC_FILTER_NAME=topic-filter
     TOPIC_FILTER_SUBSCRIPTION_NAME=topic-filter-subscription
     TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME=topic-filter-default-subscription
-
     ```
 
+## Setup
+
+Go through the following setup inorder to delete and create the required servicebus-entities(just before running each test) and for authenticating to the servicebus using `AadTokenCredentials` instead of `ConnectionString`.
+
+_**Note:** This setup is **required** only if you want to clean-up the servicebus-entities before running each test `or` to work on tests that uses `AadTokenCredentials`._
+
+**Register a new application in AAD**
+
+- Follow the [Docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) and register a new application in the Azure Active Directory(in the azure-portal).
+- Note down the `CLIENT_ID` and `TENANT_ID`.
+- Create a `SECRET`(in the Certificates & Secrets tab) and note that down.
```

Comment:
Do we need more details on how to create the secret?
If not then change this to, `In the "Certificates & Secrets" tab, create a secret and note that down"